package.json

```diff
@@ -13,15 +18,15 @@
   ],
   "repository": {
     "type": "git",
-    "url": "http://github.com/aexmachina/factory-girl.git"
+    "url": "https://github.com/celsomarques/factory-girl-amd"
   },
   "bugs": {
-    "url": "http://github.com/aexmachina/factory-girl/issues"
+    "url": "https://github.com/celsomarques/factory-girl-amd/issues"
   },
   "licenses": [
     {
       "type": "MIT",
-      "url": "http://github.com/aexmachina/factory-girl/raw/master/LICENSE"
+      "url": "https://github.com/celsomarques/factory-girl-amd/raw/master/LICENSE"
     }
   ],
   "dependencies": {},
```

Comment:
Sorry, that's because at the first time I forked your project I changed a lot of things, but I realized the best option is to make a pull request and separate the Backbone part.
And sorry about indentation, I'll look my Vim configuration.